ssn/integrated/informativeExamples/InkBird-IBS-TH2.ttl

```diff
@@ -0,0 +1,134 @@
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix qudt: <http://qudt.org/schema/qudt/> .
+@prefix unit: <http://qudt.org/vocab/unit/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix schema: <http://schema.org/> .
+@prefix gs1: <https://gs1.org/voc/> .
+@prefix equipment: <https://rdf.ag/o/equipment/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix sosa: <http://www.w3.org/ns/sosa/> .
+@prefix ssn: <http://www.w3.org/ns/ssn/> .
+@prefix ssn-system: <http://www.w3.org/ns/ssn/systems/> .
+@base <https://rdf.ag/o/equipment/> .
+
+# This example is a specification for a common over-the-shelf temperature and humidity sensor. It is a specification
+# representing the commercial sensor product from which different instantions can be derived for each specific physical instance of
+# the product.
+#
+# This prevents duplication of the information required for each new physical instance while allowing results to be related to the
+# actualy physical device that generated them.
+# 
+
+<InkBird-IBS-TH2> a owl:Class, gs1:Product ;
+    rdfs:label "Inkbird IBS-TH2"@en ;
+    rdfs:subClassOf sosa:Platform, equipment:Equipment, <https://w3id.org/seas/BluetoothCommunicationDevice> ;
```

Comment:
I am not familiar with gs1, but I think this class should be declared as a sub-class of gs1:Product, not as an instance of gs1:Product.

Reply:
Agreed.